pkg/controller/v1beta1/inferenceservice/reconcilers/cabundleconfigmap/cabundle_configmap_reconciler_test.go

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright 2021 The KServe Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cabundleconfigmap
+
+import (
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/kserve/kserve/pkg/constants"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func TestGetDesiredCaBundleConfigMapForUserNS(t *testing.T) {
+	cabundleConfigMapData := make(map[string]string)
+
+	// cabundle data
+	cabundleConfigMapData["cabundle.crt"] = "SAMPLE_CA_BUNDLE"
+	targetNamespace := "test"
+	testCases := []struct {
```

Comment:
Not a blocking comment:
   Although I see this is following the pattern of other files, since there is a single testCase, for legibility, I'd suggest to re-write as a plain test (i.e. remove the `for` loop)

Reply:
I will keep this way for future possible test cases.